src/frontend/utils/dato.tsx

```diff
@@ -41,7 +41,10 @@ export const validerDato = (
     sluttdatoAvgrensning = ''
 ): FeltState<string> => {
     if (feltState.verdi === '') {
-        return feil(feltState, <SpråkTekst id={feilmeldingSpråkId} />);
+        return feil(
+            feltState,
+            feilmeldingSpråkId !== '' ? <SpråkTekst id={feilmeldingSpråkId} /> : 'feil'
```

Comment:
Burde være tom streng i stedet for feil 🤔 når kan denne komme inn som tom streng?

Reply:
Når som helst egentlig, vi har bare sagt at feilmeldingSpråkId skal være en streng, ikke noen restrictions på lengde. Grunnen til at jeg satte den til 'feil' her er mest for at man skal oppdage det hvis det faktisk kommer inn en tom id her.